settings/R/write.settings.R

```diff
@@ -0,0 +1,34 @@
+##' Takes in a settings object, performs a series of checks, fixes & updates settings and produces pecan.CHECKED.xml
+##'
+##' @title Fix Deprecated Settings
+##' @param settings settings list
+##' @return updated settings list
+##' @author Ryan Kelly
+##' @export write.settings
+
+
+write.settings <- function(settings, outputfile = "pecan.CHECKED.xml"){
+  library(XML)
+  library(PEcAn.DB)
+  library(PEcAn.utils)
```

Comment:
1) Can you think of a more appropriate name for this function? write.settings sounds like a function that reads in a settings object and writes it to file. This does a lot more than that!  Unfortunately clean.settings and update.settings are already in use. 
2) packages